### PR TITLE
Display visitor restrictions

### PR DESCRIPTION
**Pull request by chrishanes**
This shows a list of restrictions against each visitor.

Awaiting the restrictions feature flag to show/hide visitor restrictions as well as prisoner restrictions.

### CODE REVIEW

File: app/assets/javascripts/modules/moj.matchVisitors.js [hunk 1]
```diff
@@ -211,6 +218,10 @@
       return isBanned;
     },
 
+    isContactRestricted: function(contact) {
+      return contact && contact.restrictions && contact.restrictions.length > 0? true : false;
```
Comment (TobyRet) [inline app/assets/javascripts/modules/moj.matchVisitors.js]: @chrishanes - if / else statement  not required here? ie just return bool statement

File: app/services/staff_nomis_checker.rb [hunk 2]
```diff
@@ -56,7 +56,7 @@ def offender
   end
 
   def prisoner_restrictions
-    if Nomis::Feature.offender_restrictions_info_enabled?(@visit.prison_name) &&
+    if Nomis::Feature.restrictions_info_enabled?(@visit.prison_name) &&
         offender.valid?
```
Comment (TobyRet) [inline app/services/staff_nomis_checker.rb]: Perhaps outside the scope of this PR but wonder if we need consistency around terms 'prisoner' vs 'offender' - it feels like we are randomly using one or the other

File: docs/configuration.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ A comma separated list of prisons for which PVB2 API will return slots along wit
 are available at the requested prison.
 
 
-### `NOMIS_STAFF_OFFENDER_RESTRICTIONS_ENABLED`
+### `NOMIS_STAFF_RESTRICTIONS_ENABLED`
```
Comment (TobyRet) [inline docs/configuration.md]: Is the use of 'staff' or potentially redundant or misleading?  The restrictions relate to an offender or visitor  ... Perhaps we no longer need to refer to 'staff'

Comment (StupidCodeFactory) [reply docs/configuration.md]: It's convinient to have STAFF in env var names as some of the env vars are to turn on flags for the public site

Comment (TobyRet) [reply docs/configuration.md]: Cool

File: app/assets/javascripts/modules/moj.matchVisitors.js (hunk 1 above)
Comment (chrishanes) [reply app/assets/javascripts/modules/moj.matchVisitors.js]: What do you mean @TobyRet ?

Comment (StupidCodeFactory) [reply app/assets/javascripts/modules/moj.matchVisitors.js]: I think you could drop the `? true : false` as what's before will return true or false

Comment (StupidCodeFactory) [reply app/assets/javascripts/modules/moj.matchVisitors.js]: i think...

Comment (chrishanes) [reply app/assets/javascripts/modules/moj.matchVisitors.js]: Ah, OK

**Comment by TobyRet** [comment]
From the PR description you say you are ' ... Awaiting the restrictions feature flag to show/hide visitor restrictions'
Has this requirement been met?

File: app/services/staff_nomis_checker.rb (hunk 2 above)
Comment (TobyRet) [reply app/services/staff_nomis_checker.rb]: Will add a ticket

**Pull request merged**